pandas/tests/io/formats/test_to_latex.py

```diff
@@ -584,6 +653,31 @@ def test_to_latex_longtable_caption_label(self):
 \midrule
 \endfoot
 
+\bottomrule
+\endlastfoot
+0 &  1 &  b1 \\
+1 &  2 &  b2 \\
+\end{longtable}
+"""
+        assert result_cl == expected_cl
+
+        # test when the caption, the short_caption and the label are provided
```

Comment:
Same comment here - modular tests are much easier to debug in case of future issues

Reply:
I absolutely agree. I would prefer this https://github.com/pandas-dev/pandas/pull/36528 to be merged first, and then rebase on top of that with the separate tests.

Reply:
Splitted tests.